components/licence-title.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function LicenceTitle ({
+	displayName = null,
```

Comment:
if it makes sense to have optional displayName (does it?), default should be empty string
```suggestion
	displayName = '',
```